test/CellTest.java

```diff
@@ -0,0 +1,58 @@
+import com.sudoku.Cell;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * This class represent the unit tests for the Cell class
+ *
+ * @author Alejandra Arteaga
+ */
+public class CellTest {
+    @Test
+    public void getPosXTest() {
```

Comment:
you need to verify the default value of the cell when you dont send the value in the constructor